### PR TITLE
[bundle-size-chart] Initial version

### PR DESCRIPTION
**Pull request by danielrozenberg**
Remember that this is mostly a hack and not a serious piece of code :)

// cc: @kevinkimball

### CODE REVIEW

File: bundle-size-chart/app.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019, the AMP HTML authors
```
Comment (estherkim) [inline bundle-size-chart/app.yaml]: Should this file be uploaded to GitHub?

File: bundle-size-chart/src/app.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+const CRON_TIMEOUT_MS = 600000; // 10 minutes, the GAE cron timeout limit.
+const MAX_COMMIT_DAYS = 365;
+const GCLOUD_STORAGE_BUCKET = 'amp-bundle-size-chart';
+
+dotenv.config();
+
+const app = express();
+app.use(express.static('static'));
+
+app.get('/_cron', async (request, response) => {
```
Comment (estherkim) [inline bundle-size-chart/src/app.ts]: Would like to see methods split out here with corresponding unit tests. Not needed right away.

Comment (danielrozenberg) [reply bundle-size-chart/src/app.ts]: I very much doubt I'll ever write tests for this 😅

File: bundle-size-chart/app.yaml (hunk 1 above)
Comment (danielrozenberg) [reply bundle-size-chart/app.yaml]: Yeah, there's nothing secret about this file. I dunno why I excluded it from the other apps...

**Pull request merged**